### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by aliuk2012**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    echo "running create"
+    bundle exec rails db:setup
+    ;;
+migrate)
+    echo "running migrate and seed"
+    bundle exec rails db:migrate
+    ;;
+esac
+bundle exec puma -d -C config/puma.rb
+tail -f /var/log/dmesg
```
Comment (niallcreech) [inline run.sh]: you sure you want to tail the dmesg logs in run.sh? whats the *best* logs you'd want out of a running docker container?

Comment (misaka) [inline run.sh]: 👍 ... where are the application/puma logs here?

Comment (aliuk2012) [inline run.sh]: @niallcreech Is there a standard approach to this? Just looking at PVB-2 they seem to tail the dmesg logs in run.sh. https://github.com/ministryofjustice/prison-visits-2/blob/master/run.sh

Comment (niallcreech) [inline run.sh]: PVB are shipping their app logs. You should have something similar in your application directory when you run. something like log/production.log

**Pull request merged**